webapp/app/[locale]/stake/page.tsx

```diff
@@ -29,7 +29,10 @@ const PageBackground = () => (
 )
 
 export default function Page() {
-  const stakeTokens = useStakeTokens()
+  // Removing WETH from the list of tokens to stake
+  // here instead of in the tokenList file
+  // It has to be rendered on dashbord page though
+  const stakeTokens = useStakeTokens().filter(t => t.symbol !== 'WETH')
```

Comment:
if it is a token that can be staked, but hidden/merged from the dashboard, then I would keep `WETH` as part of the hook.

```suggestion
  // Removing WETH from the list of tokens to stake
  // here instead of in the tokenList file
  // It has to be rendered on dashbord page though
  const stakeTokens = useStakeTokens()
```

Reply:
@gndelia not sure if I understood well, but we just need to filter from the strategy table.. if I move this filter inside the hook, it would affect the dashboard as well.

Reply:
Re-reading my comment, I think it makes no sense. Somehow I thought that line of code was inside the hook, in the implementation. (That's why I said "Let's keep WETH as part of the hook"). I guess I Was too tired already to understand what I was reading 😅

Reply:
Lol yeah I got confused reading it. No worries at all.